src/platform/plugins/shared/dashboard/common/lib/dashboard_panel_converters.ts

```diff
@@ -23,9 +23,10 @@ export const convertPanelsArrayToPanelMap = (panels?: DashboardPanel[]): Dashboa
   const panelsMap: DashboardPanelMap = {};
   panels?.forEach((panel, idx) => {
     const panelIndex = panel.panelIndex ?? String(idx);
+    const gridData = panel.gridData ?? {};
     panelsMap![panel.panelIndex ?? String(idx)] = {
       type: panel.type,
-      gridData: panel.gridData,
+      gridData: gridData.sectionId === undefined ? omit(gridData, 'sectionId') : gridData,
```

Comment:
Why are you removing sectionId when its undefined? Maybe add a comment on why this is needed?

Reply:
I figured it was better to not store undefined values 🤷 This was more of a cleanliness thing than anything so that saving a dashboard after this PR lands will not introduce any changes to the Dashboard saved objects (assuming you haven't added sections). I'm fine with removing it if we don't care about that :)

Reply:
I think its fine, just a comment so we know why this is here

Reply:
Done in e232deeef2b59972fc0084fb4dce14369cb7599d :+1: